components/LandingPageCard.vue

```diff
@@ -0,0 +1,29 @@
+<template>
```

Comment:
Can this contextualised "card" component reuse the generic `Card` component? Or can the generic `Card` component be renamed so that it is not generic?

Reply:
Yep definitely - the plan is to have the Card component be generic which could have an image or an icon etc. To get this change released though we decided not to merge them just yet and call this component the LandingPageCard